packages/core-react/index.d.ts

```diff
@@ -8,4 +8,21 @@ export type TextFieldProps = {
 
 declare const TextField: React.FC<TextFieldProps>;
 
-export { TextField };
+export type ContainerProps = {
+  Tag?: React.ElementType<any>;
+} & React.HTMLProps<HTMLElement>;
+
+declare const Container: React.FC<ContainerProps>;
+
+export type GridProps = React.HTMLProps<HTMLElement>;
+
+declare const Grid: React.FC<GridProps>;
+
+export type CellProps = {
+  push?: 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+  span?: 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+} & React.HTMLProps<HTMLElement>;;
```

Comment:
woops `;;` but I'll fix this for you!

Reply:
good catch! Surprised this wasn't caught by a commit hook